spec/rails_helper.rb

```diff
@@ -13,9 +13,10 @@
 require "fakeredis/rspec"
 require "pundit/matchers"
 require "pundit/rspec"
-require "webmock/rspec"
 require "sidekiq/testing"
+require "test_prof/factory_prof/nate_heckler"
 require "validate_url/rspec_matcher"
+require "webmock/rspec"
```

Comment:
Where did this come from? Asking only because it looks not immediately related to the task at hand.

Reply:
Just sorted the lines :D